.travis.yml

```diff
@@ -19,10 +19,15 @@ env:
     - SPARK_VERSION=2.3.0
     - SPARK_BUILD="spark-${SPARK_VERSION}-bin-hadoop2.7"
     - SPARK_BUILD_URL="https://dist.apache.org/repos/dist/release/spark/spark-2.3.0/spark-2.3.0-bin-hadoop2.7.tgz"
+    - SPARK_HOME=$HOME/.cache/spark-versions/$SPARK_BUILD
+    - RUN_ONLY_LIGHT_TESTS=True
   matrix:
-    - PYTHON_VERSION=2.7.13
-    - PYTHON_VERSION=3.5.1
-    - PYTHON_VERSION=3.6.2
+    - PYTHON_VERSION=3.6.2 TEST_SUITE=scala-tests
+    - PYTHON_VERSION=3.6.2 TEST_SUITE=python-tests
+    - PYTHON_VERSION=3.6.2 TEST_SUITE=pylint
+    - PYTHON_VERSION=3.5.1 TEST_SUITE=pylint
```

Comment:
Why are we running pylint with more python versions than python-tests?

Reply:
haha, I misunderstood that's what was required based on your comment about matrix. updated, thanks for catching this!